src/mcts/stoppers/factory.cc

```diff
@@ -43,9 +44,12 @@ const OptionId kMoveOverheadId{
     "Amount of time, in milliseconds, that the engine subtracts from it's "
     "total available time (to compensate for slow connection, interprocess "
     "communication, etc)."};
-const OptionId kTimeManagerId{"time-manager", "TimeManager",
-                              "Name and config of atime manager."};
-
+const OptionId kTimeManagerId{
+    "time-manager", "TimeManager",
+    "Name and config of a time manager. "
+    "Default is 'legacy'. Optional time manager is 'alphazero' "
```

Comment:
I still think 'alphazero' time manager gets disproportionally more attention here than 'smooth'. :-P

Also http://lczero.org/dev/docs/timemgr is a design doc for my PR, it's not an instuction to users. Better have http://lczero.org/play/configuration/flags/ and describe all parameters there (not really needed to do that before merging though).

Also maybe even better will be to have a short url, https://lc0.org/timemgr.

How about `Name and configuration of a time manager. Possible names are 'legacy' (default), 'smooth-experimental' and 'alphazero'. See https://lc0.org/timemgr for configuration details.` ?

Reply:
Updated in github editor as suggested :)